babel.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: ['module:@react-native/babel-preset'],
+  presets: ['module:react-native-builder-bob/babel-preset'],
```

Comment:
Is this necessary? Just confused on its relevance to this change

Reply:
There were quite a number of changes irrelevant to the turbomodules / new arch fix itself, that just came along with updating to the latest RN Native Module library template. I'm not certain about this one, but agree it'd be good to read up just to stay up to date with RN practices.